homeassistant/components/ecovacs/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "ecovacs",
   "name": "Ecovacs",
   "documentation": "https://www.home-assistant.io/integrations/ecovacs",
-  "requirements": ["sucks==0.9.4"],
+  "requirements": ["py-sucks==0.9.7"],
```

Comment:
Side note: Is the library still abandoned?

https://github.com/mib1185/py-sucks/blob/5e84f8c2f6c25427ae09bec5e63cc87d403888fe/setup.py#L20

Reply:
a valid point 🙈  i've removed the abandoned from the description. unfortunately, this needs a new release 😆